ui/src/pages/GlobalHome_test.js

```diff
@@ -0,0 +1,103 @@
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import toJson from 'enzyme-to-json';
+import {createMemoryHistory} from 'history';
+import React from 'react';
+import {MemoryRouter} from 'react-router';
+
+import GlobalHome from './GlobalHome';
+import {asyncMountWithIntl} from '../testing/enzyme-intl-helper';
+import {runPageTest} from '../testing/utils';
+
+Enzyme.configure({adapter: new Adapter()});
+
+async function setupPageWrapper() {
+  fetch.mockResponseOnce(JSON.stringify([
+    {repoId: 'haiti', title: 'Haiti', recordCount: 400},
+    {repoId: 'japan', title: 'Japan', recordCount: 100},
+    {repoId: 'albany', title: 'Albany', recordCount: 100},
+    {repoId: 'seattle', title: 'Seattle', recordCount: 800},
+    {repoId: 'montreal', title: 'Montreal', recordCount: 500},
+    {repoId: 'belgium', title: 'Belgium', recordCount: 100},
+    {repoId: 'boise', title: 'Boise', recordCount: 100},
+  ]));
+  const history = createMemoryHistory('/');
+  const wrapper = await asyncMountWithIntl(
+    <MemoryRouter>
+      <GlobalHome history={history} />
+    </MemoryRouter>
+  );
+  return [wrapper, history];
+}
+
+describe('testing GlobalHome', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+
+  test('repo cards are rendered', async (done) => {
```

Comment:
The `async` and `done` are generally mutually exclusive.

If you're not testing anything asynchronous, you don't need the test method to `async` and you don't need done. Return nothing from the method and the framework will assume the test has completed when the method ends.

If you're testing async stuff (as in this case), you can:

1. Return a promise which will resolve when all async behaviour and assertions have been made. The framework will know the test is done when the promise has completely resolved.
2. Make the test method `async`. The framework will assume the test is done when the async method completes. (This is equivalent to the above).
3. Accept the `done` param in its signature and call it when the test is done. In most cases, you don't need this because 1 or 2 are sufficient, but there are edge cases where this is still handy (e.g. the application code is hiding the async behaviour, but you're able to pass it the `done` as some kind of completing callback).

Reply:
I ended up going back to using done. I'm not sure it's worth it to include the dependencies I would need to deal with the regeneratorRuntime stuff and support async/await.